src/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```

Comment:
What are your plans about this? (it's seems dangerous)

Reply:
i'dont know. I can research about this service. it's add automatically when install react

Reply:
Service worker is for PWA.